src/sentry/data/config/config.yml.default

```diff
@@ -57,6 +57,21 @@ redis.clusters:
       0:
         host: 127.0.0.1
         port: 6379
+  rc-default:
```

Comment:
is this enough to use it? or does one need to actually put `rc-default` somewhere else to use it?

Reply:
This adds another Redis cluster. You need to configure some service to use it.